.editorconfig

```diff
@@ -0,0 +1,14 @@
+# Normalizes editor configuration.
```

Comment:
this file is not related to this PR. Would be best in another PR

Reply:
I will keep it in this PR.